VSCODE.md

```diff
@@ -0,0 +1,58 @@
+# Starting Development with Visual Studio Code
+
+You can get started with development very quickly with Visual Studio Code amd Docker.
+
+## Install Prerequisites
+
+* Install Docker.
+* Install VS Code and install the "Remote - Containers" extension in VS Code.
+* Configure your Docker resources to give sufficient resources to the
```

Comment:
This is very platform dependent (e.g. on Linux docker by default has access to all resources of the host machine), but on Docker Desktop on Mac, you can tweak via UI preferences. On Windows, using WSL2 as the docker backend (that's the default on current versions), you don't have settings for this so nothing to configure as far as I'm aware.

Windows using Hyper-V (now considered legacy) does provide knobs and dials for this, I believe.

So perhaps simplest to make this note Mac-specific to avoid Windows users hunting for something they won't find?